airflow/dags/gtfs_static_downloader.py

```diff
@@ -0,0 +1,113 @@
+"""
+Download the state of CA GTFS files, async version
+"""
+import intake
+import requests
+import logging
+import zipfile
+import io
+import pathlib
+import datetime
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+
+# TODO: Fix to Pathlib
+catalog = intake.open_catalog("./dags/catalogs/catalog.yml")
+
+
+def make_gtfs_list():
+    """
+    Read in a list of GTFS urls
+    from the main db
+    plus metadata
+    """
+    df = catalog.official_list(
+        csv_kwargs={
+            "usecols": ["ITP_ID", "GTFS", "Agency Name"],
+            "dtype": {"ITP_ID": "float64"},
+        }
+    ).read()
+    # TODO: Figure out what to do with Metro
+    # For now, we just take the bus.
+
+    # TODO: Replace URLs with Zip ones.
+    # For now we filter, and then remove ones that don't contain
+    # zip filters
+    df = df[(df.GTFS.str.contains("zip")) & (df.GTFS.notnull())]
+    return df
+
+
+def clean_urls(url):
+    """
+    take the list of urls, clean as needed
```

Comment:
note use as a `pd.apply` - doesn't take a list, takes an individual

Reply:
renaming and updating docs.